app/assets/stylesheets/application.css

```diff
@@ -55,34 +73,42 @@ a:focus {
 }
 
 .header {
-  background-color: var(--black);
-  padding: 2em;
+  background-color: rgba(15,15,15,1);
+  padding: 1.25em;
 }
 
 .header-item {
   display: inline-block;
 }
 
-.header-logo {
-  width: 373px;
+.header-title {
+  color: white;
+  font-family: "Fira-Bold", Helvetica, Arial, sans-serif;
+  font-size: 21px;
+  font-weight: bold;
 }
 
 .header-login {
-  color: var(--white);
+  color: rgba(208,205,205,1);
   float: right;
   vertical-align: middle;
+  font-size: 16px;
+  font-family: "Fira", Helvetica, Arial, sans-serif;
 }
 
 .header-logout-link {
-  border: 2px solid var(--white);
-  border-radius: 5px;
-  color: var(--white) !important;
+  border-radius: 4px;
+  color: rgba(208,205,205,1) !important;
```

Comment:
@JacOng17 Just wondering if you know why we are using important here? What css rule is it overriding?

Reply:
Nope, that was the original code

Reply:
Removed it and it doesn't seem to affect anything...